homeassistant/components/zha/core/discovery.py

```diff
@@ -184,15 +192,33 @@ def _async_handle_single_cluster_matches(hass, endpoint, zha_device,
             ))
 
     for cluster in endpoint.out_clusters.values():
+        if cluster.cluster_id in OUTPUT_CHANNEL_ONLY_CLUSTERS:
+            cluster_match_results.append(
+                _async_handle_channel_only_cluster_match(
+                    zha_device,
+                    cluster,
+                    is_new_join,
+                ))
+            continue
+
+        device_type = cluster.endpoint.device_type
+        profile_id = cluster.endpoint.profile_id
+
         if cluster.cluster_id not in profile_clusters:
-            cluster_match_results.append(_async_handle_single_cluster_match(
-                hass,
-                zha_device,
-                cluster,
-                device_key,
-                SINGLE_OUTPUT_CLUSTER_DEVICE_CLASS,
-                is_new_join,
-            ))
+            # prevent remotes and controllers from getting entities
+            if not (cluster.cluster_id == OnOff.cluster_id and profile_id in
```

Comment:
Since binary sensors for "remotes" are gone, are there any other "output clusters" for which we have to create entities?

Reply:
yes, we do. Ikea motion sensors, HUE and likes.